lambdatimes/src/components/Header.js

```diff
@@ -1,16 +1,58 @@
 import React from 'react';
-
-// Refactor this component to use styled components and not classNames. 
-// You can find the corresponding CSS in the CSS/index.css file
+import Style from 'styled-components';
 
 const Header = () => {
   return (
-    <div className="header">
-      <span className="date">SMARCH 32, 2018</span>
+    <HeaderDiv>
+      <DateSpan>SMARCH 32, 2018</DateSpan>
       <h1>Lambda Times</h1>
-      <span className="temp">98°</span>
-    </div>
+      <TempSpan>98°</TempSpan>
+    </HeaderDiv>
   )
 }
 
-export default Header
\ No newline at end of file
+export default Header
+
+ /* Component Styles */
+
+const HeaderDiv = Style.div`
```

Comment:
Great use of styled components - how useful are you finding the styled components and can you see a use for them in projects that you may do in the future